ComponentKit/Core/Scope/CKComponentScopeHandle.h

```diff
@@ -30,6 +30,7 @@
 
 /** Creates a conceptually brand new scope handle */
 - (instancetype)initWithListener:(id<CKComponentStateListener>)listener
+                globalIdentifier:(CKComponentScopeHandleIdentifier)globalIdentifier
```

Comment:
wait if it's conceptually "brand new", why are you passing a global identifier *in* instead of letting it be allocated by the callee?